src/main/java/lotto/domain/PassiveLottoMaker.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PassiveLottoMaker implements LottoMaker {
+
+    private static final String SPLIT_TOKEN = ",";
+
+    private final List<String> lottoNumbers;
+    public PassiveLottoMaker(List<String> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    @Override
+    public List<Lotto> create() {
+        return lottoNumbers.stream()
+                .map(numbers -> Arrays.stream(numbers.split(SPLIT_TOKEN))
+                        .mapToInt(Integer::parseInt)
+                        .mapToObj(LottoNumber::from)
```

Comment:
정적 팩토리 함수를 사용한다면 LottoNumber의 매개변수로 문자열을 받아서 처리해주도록 해도 될 것 같아요! 
```suggestion
                        .mapToObj(LottoNumber::from)
```